vxtelegram/telegram.py

```diff
@@ -177,6 +196,23 @@ def handle_raw_inbound_message(self, message_id, request):
         )
         request.finish()
 
+    @inlineCallbacks
+    def is_duplicate(self, update_id):
+        """
+        Checks to see if an incoming update has already been processed
+        """
+        exists = yield self.redis.exists(update_id)
```

Comment:
I think we should namespace the redis key name a bit here. E.g. write a function like:
```python
def update_id_key(self, update_id):
    return "updates:%s" % (update_id,)
```
and then do:
```
update_key = self.update_id_key(update_id)
exists = yield self.redis.exists(update_key)
```
This means if we want to add more kinds of keys later, we don't need to worry about name clashes.